fastlane/lib/fastlane/fastlane_require.rb

```diff
@@ -43,9 +43,19 @@ def gem_installed?(name, req = Gem::Requirement.default)
         # don't actually load anything we don't want to load
         # This is just to test if the gem is already preinstalled, e.g. YAML
         # See https://github.com/fastlane/fastlane/issues/6951
-        spawn do
+        fork do
           begin
             require name
+          rescue NotImplementedError => e
+            puts e
+            # fork method is not available (Windows?)
+            spawn do
```

Comment:
for spawn could you try this : 

pid = spawn(RbConfig.ruby, "-erequire 'Spaceship'") or something of the kind?

Reply:
Yes, probably. Right now I don't really understand how to trigger and test this code, so I will probably just "deactivate" it for platforms that don't support `fork` (in another PR, I will close this one) and see if we can fix this further down the line.